doc/manual/config-basic.rst

```diff
@@ -40,6 +40,22 @@ A jury or administrative user can also be associated with a team. This
 will enable that user to submit solutions to the system, or resubmit
 edited team solutions.
 
+Resetting the password for a user
+---------------------------------
+
+If you do not have access anymore to any admin user, you can use the following
+command to reset the password of a user to a random value::
+
+  webapp/bin/console domjudge:reset-user-password admin
+
+Replace ``admin`` with the username of the user you want to reset the password for.
+The password will be displayed.
+
+If you are running DOMjudge ``< 7.2``, see
```

Comment:
Do we want to add this kind of stuff to the manual? The manual is for the version it ships with. Esp since we now have multiple doc versions also on the website. Just select your version and the docs will match.

Reply:
Hmm true better not to have it